cirq-superstaq/cirq_superstaq/service.py

```diff
@@ -165,6 +173,46 @@ def _resolve_target(self, target: Union[str, None]) -> str:
 
         return target
 
+    def _validate_cirq_circuit(self, circuits: Union[cirq.Circuit, Sequence[cirq.Circuit]]) -> None:
+        """Validates that the input is either a single cirq.Circuit or a list of cirq.Circuit
```

Comment:
```suggestion
        """Validates that the input is either a single `cirq.Circuit` or a list of `cirq.Circuit`
```
(elsewhere as well - code snippets should be wrapped in \`backticks\`)